/ros2_kdl_client_node:
  ros__parameters:
    cmd_interface: "position"
    traj_type: "linear"
    s_type: "trapezoidal"

    traj_duration: 15.0
    acc_duration: 1.0
    total_time: 20.0
    trajectory_len: 50.0
    Kp: 10.0
    end_position: [0.45, 0.2, 0.55]

    #ctrl: "velocity_ctrl"